etc/systemd/system/zfs-scrub-monthly@.timer.in

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Monthly zpool scrub timer for %i
+Documentation=man:zpool-scrub(8)
+
+[Timer]
+OnCalendar=monthly
+AccuracySec=1h
```

Comment:
Isn't one hour overkill? If I'm reading the docs right, this is meant to coalesce timers that fire too quickly. I know it can be changed, but the default of one minute seems more reasonable.

Reply:
Right, it should be something like this:
```
RandomizedDelaySec=1h
AccuracySec=1s
```
(or `AccuracySec=1m` would be fine too)

Reply:
`AccuracySec=1m` (the default) seems reasonable.

On my system, I'd probably remove the delay -- I like my scrubs to start at a predictable time, and one hour is too much.